src/sagas/admin.js

```diff
@@ -53,17 +81,24 @@ function sendTxWrapper({ txObject, gasPrice, gasLimit, fromAddress }) {
       (err, res) => {
         // callback passed into eth.contract.send() to get the txhash
         if (err) {
-          reject(err);
+          if (
+            err.message ===
```

Comment:
Where does this message comes from ? It looks like a weak comparison (i.e may easily break)

Why not always displaying the toaster `toast.error("Transaction was rejected.");`?

Reply:
err is returned message from Web3 and not all error messages are caused by rejected transactions. So it is necessary to only show the message when the transaction has actually been rejected.

Reply:
does this handle ledger as well ?

Reply:
as per @rjchow :bulb: 

let's use a different check: `err.message.includes('User denied transaction signature.')`
Some sources
https://ethereum.stackexchange.com/questions/45836/elegant-way-to-map-web3-js-error-messages#comment54609_45848
https://github.com/blocknative/assist/blob/afbf565dda006158f6f1397d8ea645fa6d307eff/src/js/helpers/utilities.js#L134

What about ledger ?

Reply:
Updated and works for ledger too in latest commit.

Reply:
Please add a comment why we are doing this, it's not obvious at all =)